src/libmongoc/tests/test-libmongoc.c

```diff
@@ -499,6 +499,16 @@ test_framework_getenv (const char *name)
    return _mongoc_getenv (name);
 }
 
+char *
+test_framework_getenv_required (const char *name)
```

Comment:
+1, nice! (Although, do we ever call it?)

Reply:
Yes, this is called in `test-mongoc-client-side-encryption.c` when reading required environment variables.